packages/vendor-core/lib/modules.js

```diff
@@ -9,7 +9,8 @@ module.exports = [
   /**
    * React related
    */
-  'babel-polyfill',
+  'core-js',
+  'regenerator-runtime',
```

Comment:
Based on the migration guide, shouldn't it suppose to be?
```js
'core-js/shim',
'regenerator-runtime/runtime',
```

Notice this list act as the javascript import statement.